sdcm/nemesis.py

```diff
@@ -2218,8 +2217,9 @@ def delete_half_partition(self, ks_cf):
     def delete_by_range_using_timestamp(self, ks_cf: str):
         self.log.debug('Delete by range - using timestamp')
 
-        partitions_for_delete = self.choose_partitions_for_delete(self.tester.partitions_attrs.max_partitions_in_test_table, ks_cf,
-                                                                  with_clustering_key_data=False)
+        partitions_for_delete = self.choose_partitions_for_delete(
+            partitions_amount=self.tester.partitions_attrs.non_validated_partitions // 5, ks_cf=ks_cf,
```

Comment:
I would see how this 5 is not duplicated, and defined once.
or even defined as part of the configuration, so the user can control how big the deletion are gonna big

Reply:
fixed: `NUM_DELETIONS_FACTOR = 5`